adapters/repos/db/vector/hnsw/delete_test.go

```diff
@@ -34,20 +34,20 @@ func TestDelete_WithoutCleaningUpTombstones(t *testing.T) {
 			MaximumConnections:    30,
 			EFConstruction:        128,
 			DistanceProvider:      distancer.NewCosineProvider(),
-			VectorForIDThunk: func(ctx context.Context, id int32) ([]float32, error) {
+			VectorForIDThunk: func(ctx context.Context, id int64) ([]float32, error) {
```

Comment:
whole file: see above, all node ids should be represented by `uint64`, removing the need for the conversions.